sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -1666,14 +1616,19 @@ public static ContactDto toDto(Contact source) {
 		return target;
 	}
 
+	@Override
+	public ContactReferenceDto toRefDto(Contact contact) {
+		return convertToReferenceDto(contact);
```

Comment:
Rename this to `toReferenceDto`. With this we are then finally consistent with the rest of the codebase.